src/loaders/scraper_browse_id.py

```diff
@@ -57,14 +62,18 @@ def run():
         save_browse_page(page.content, pagenum)
         id_list.extend(extract_ids(page))
     print('Done')
-    
+   
     # Save ids to file
     filepath = f'{config.DATA_PATH}/processed/game_ids.csv'
```

Comment:
Same as above: Reduce `filepath` to just the filename and combine the two when we need the full path